example/plugins/c-api/secure_link/secure_link.c

```diff
@@ -125,7 +125,7 @@ TSRemapDoRemap(void *ih, TSHttpTxn rh, TSRemapRequestInfo *rri)
   EVP_MD_CTX_destroy(ctx);
 #endif
   for (i = 0; i < MD5_DIGEST_LENGTH; i++) {
-    sprintf(&hash[i * 2], "%02x", md[i]);
+    snprintf(&hash[i * 2], 32, "%02x", md[i]);
```

Comment:
```
wkaras /tmp/x
O$ cat x.cc
char x[3][32];

void f(int i)
{
  static_assert(sizeof(x[i]) == 32);
}
wkaras /tmp/x
O$ gcc -c -pedantic -Wall -Wextra -std=c++17 x.cc
wkaras /tmp/x
O$
```
Strongly recommend using `sizeof(hash[i * 2])` instead of `32`.

Reply:
I don't understand this one. Do we have another `hash`? I only find `char hash[32]`.

Reply:
It was wrong in either way. Fixed.